docs/toc.yml

```
@@ -1338,6 +1338,9 @@ items:
     displayName: Excel
 - name: Resources
   items:
+  - name: Office Add-ins glossary
```

Since it's in the Office Add-ins documentation, it might suffice to simply say "Glossary"
```suggestion
  - name: Glossary
```